XamarinMoviesApp/XamarinMoviesApp.UITests/Tests.cs

```diff
@@ -0,0 +1,46 @@
+﻿using NUnit.Framework;
+using Xamarin.UITest;
+using XamarinMoviesApp.UITests.Pages;
+
+namespace XamarinMoviesApp.UITests
+{
+    [TestFixture(Platform.Android)]
+    [TestFixture(Platform.iOS)]
+    public class Tests
+    {
+        IApp app;
+        Platform platform;
+
+        public Tests(Platform platform)
+        {
+            this.platform = platform;
+        }
+
+		[SetUp]
+		public void BeforeEachTest()
+		{
+			app = AppInitializer.StartApp(platform);
+		}
+
+        [Test]
+        public void VisualizationTest()
+        {
+            new MoviesPage()
+                .SelectFirstMovie();
+
+            new MovieDetailPage()
+                .GoBack();
+        }
+
+		[Test]
+		public void SearchTest()
+		{
+            new MoviesPage()
+                .SearchMovie("Star Wars")
+                .SelectFirstMovie();
+
+			new MovieDetailPage()
+				.GoBack();
```

Comment:
Sure. This UITest was first intended to verify some interactions with the app, but it is better check the result thought asserts. 

I have made some improvements to the project as you suggested and there is a new commit with those changes.